src/common/Configuration.h

```diff
@@ -65,7 +65,7 @@ namespace SDDM {
             Entry(SessionDir,          QString,     _S("/usr/share/xsessions"),                 _S("Directory containing available X sessions"));
             Entry(SessionCommand,      QString,     _S(SESSION_COMMAND),                        _S("Path to a script to execute when starting the desktop session"));
 	    Entry(SessionLogFile,      QString,     _S(".local/share/sddm/xorg-session.log"),   _S("Path to the user session log file"));
-	    Entry(UserAuthFile,        QString,     _S(".Xauthority"),                          _S("Path to the Xauthority file"));
+	    Entry(UserAuthFile,        QString,     _S(".local/share/sddm/.Xauthority"),        _S("Path to the Xauthority file"));
```

Comment:
Instead of hardcoding `~/.local/share/` - shouldn't this be determined through `QStandardPaths::DataLocation`?

Reply:
It's the same directory as the logfile location entry, which is hardcoded as well.

Reply:
I mean that the default value is hardcoded.